common/beerocks/bwl/include/bwl/ap_wlan_hal.h

```diff
@@ -311,13 +311,21 @@ class ap_wlan_hal : public virtual base_wlan_hal {
     virtual bool read_acs_report() = 0;
 
     /*!
-     * Read the supported channls from the hardware
+     * Read the supported channels from the hardware
      * On successful completion the information can be retrieved 
      *
      * @return true on success or false on error.
      */
     virtual bool read_supported_channels() = 0;
 
+    /*!
+     * Copy the supported channels  to the preferred channels 
+     * On successful completion the preferred channels  can be retrieved 
+     *
+     * @return true on success or false on error.
+     */
+    virtual bool update_preference_channels_from_supported_channels() = 0;
```

Comment:
So this API and this whole commit should be dropped.

Reply:
removed the commit that adds `update_preference_channels_from_supported_channels`
and in its stead changed the name of the `read_supported_channels()` to `read_preferred_channels()`

If the ACS is disabled the `read_preferred_channels()` will be used to fill the `preferred_channels`